packages/components/psammead-inline-link/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0 | [PR#1365](https://github.com/bbc/psammead/pull/1365) Bump psammead-styles to 1.2.0 |
```

Comment:
Why are we bumping from `alpha` to `stable` version

Reply:
! Great spot, thanks!